src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -395,11 +395,14 @@ Tuis id ullamcorper lectus, feugiat dapibus lectus. In hac habitasse platea dict
 "InviteFriends_ShareUrl" = "https://www.corona-warn-app.de";
 
 //Reset View
-"Reset_NavTitle" = "Anwendung zurücksetzen";
+"Reset_NavTitle" = "Zurücksetzen";
 "Reset_Header1" = "Sind Sie sicher, dass Sie die Anwendung zurücksetzen wollen?";
-"Reset_Descrition1" = "Hier kommt ein Erklärungstext, der beschreibt was passiert wenn man die Anwendung zurücksetzt";
+"Reset_Descrition1" = "Sie werden nicht mehr über Ihre Risiko-Begegnungen informiert und können andere Nutzerrinen und Nutzer nicht mehr warnen.";
 "Reset_Button" = "Anwendung zurücksetzen";
 "Reset_Discard" = "Abbrechen";
+"Reset_InfoTitle" = "Begegnungs-Aufzeichnung löschen";
+"Reset_InfoDescription" = "Die Begegnungs-Aufzeichnung muss separat in den Geräte Einstellungen elöscht werden.";
```

Comment:
fix typo please -> gelöscht

Reply:
This is not so important since it's gonna be changed be the Localization team anyway. But, for the sake of consistency, changed [here](https://github.com/corona-warn-app/cwa-app-ios/pull/134/commits/50647c96aef4fd956263193156dc2d0f564de0b9).